src/index.js

```diff
@@ -22,32 +51,37 @@ export default function css(options = {}) {
         }
       }
 
-      // Track the order that each stylesheet is imported.
-      if (!order.includes(id)) {
-        order.push(id)
-      }
-
       // Keep track of every stylesheet
       // Check if it changed since last render
+      // NOTE: If we are in transform block, we can assume styles[id] !== code, right?
       if (styles[id] !== code && (styles[id] || code)) {
         styles[id] = code
-        changes++
+        hasChanged = true
       }
 
       return ''
     },
     generateBundle(opts, bundle) {
-      // No stylesheet needed
-      if (!changes || options.output === false) {
-        return
+      const ids = []
```

Comment:
By returning early here, we can skip the import order determination.
```suggestion
      if (!changes || options.output === false) {
        return
      }
      const ids = []
```

Reply:
The reason I removed this is because `changes` (now `hasChanged`) is not always true when we reach this step. `transform` is only called on files that have observed changes in the file system. If the order of imports changes in a particular file, the only file that `transform` gets called on is that file, not the css files (or other) files that are being imported.

Reply:
OK! (I think however that transform is also called also when saving a file without modifications, but that's a minor edge case)